system/jlib/jlog.cpp

```diff
@@ -1289,15 +1285,15 @@ void LogMsgPrepender::report(const LogMsgCategory & cat, const char * format, ..
     buff.append(file).append("(").append(line).append(") : ").append(format);
     va_list args;
     va_start(args, format);
-    queryLogMsgManager()->report_va(cat, unknownJob, buff.str(), args);
+    queryLogMsgManager()->report_va(cat,  buff.str(), args);
```

Comment:
trivial/formatting: double space

Reply:
Yes, an unfortunate side-effect of a global search and replace that didn't include the trailing space.  I'll try and fix.